app/models/invenio_rdm_record_converter.rb

```diff
@@ -12,14 +12,18 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   ENGLISH = "english"
   CREATOR_UNKNOWN = 'unknown'
   CREATOR_NOT_IDENTIFIED = "creator not identified."
+  MEMOIZED_PERSON_OR_ORG_DATA_FILE = 'memoized_person_or_org_data.txt'
+  ROLE_OTHER = 'Other'
```

Comment:
Needs to be lowercase
```suggestion
  ROLE_OTHER = 'other'
```

Reply:
This is outdated. Github must have saved the comment before the change was updated.